packages/sanity/src/desk/comments/src/components/pte/config.ts

```diff
@@ -0,0 +1,34 @@
+import {Schema} from '@sanity/schema'
+import {defineArrayMember, defineField} from '@sanity/types'
+
+export const mentionObject = defineField({
+  name: 'mention',
+  type: 'object',
+  fields: [
+    {
+      name: 'userId',
+      type: 'string',
+    },
+  ],
+})
+
+const blockType = defineField({
```

Comment:
Consider adding this, denying decorators and annotations from pasted content.

```
  marks: {
    annotations: [],
    decorators: [],
  },
```

What about links? Should they be allowed to be pasted?

Reply:
Nope, since we don't render them! I think this closes [EDX-554](https://linear.app/sanity/issue/EDX-554/pasted-links-will-display-default-annotations-at-author-time)?